.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+on:
+  release:
+    types: [created]
+
+jobs:
+  releases-matrix:
+    name: Release
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # build and publish in parallel: linux/386, linux/amd64, linux/arm64, darwin/amd64, darwin/arm64
+        goos: [linux, darwin]
+        goarch: ["386", amd64, arm64]
```

Comment:
Just out of curiosity - where would be want linux/386? That would be 32bit right? CI runners? I certainly can't imagine any mac devs using 32bit anymore...

Reply:
I was thinking of CI runners indeed, but maybe we can get rid of it

Reply:
I added the universal build to the Makefile in https://github.com/sketch-hq/gql-lint/pull/5 thinking we will be using make to build it here. It is not a hard requirement as we could get 2 binaries and put them together by hand when updating the build in Xcode